docs/_articles/en/getting-started/recommended-extensions.md

```diff
@@ -17,7 +17,7 @@ Keeping your code consistent and nicely formatted is important in any programmin
 
 <https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode>
 
-> NOTE: Unfortunately, Prettier doesn’t support Apex (a member of the community is [working on it](https://github.com/dangmai/prettier-plugin-apex) though).
+> NOTE: Prettier has support to Apex through a [plugin](https://github.com/dangmai/prettier-plugin-apex) developed by the community.
```

Comment:
@hbc-bindu - should it be? "Prettier supports Apex through a plug-in developed by the community."

Reply:
@ekapner , we could say 
Prettier also supports Apex through the [Prettier Apex plugin](https://github.com/dangmai/prettier-plugin-apex).
I think we don't have to mention _developed by the community_ because it's true for all the extensions in this article.

Reply:
Both suggestion look good to me, I will go with what @hbc-bindu is saying. Thanks for reviewing, let me know if this reads better.

Reply:
I agree, thanks @hbc-bindu and @jefersonchaves